src/redux/books/books.jsx

```diff
@@ -0,0 +1,26 @@
+const ADD = './redux/ADD';
+const DELETE = './redux/DELETE';
+
+export default function bookReducer(state = [], action) {
+  switch (action.type) {
+    case ADD:
+      return [
+        ...state,
+        {
+          id: action.id,
+          name: action.name,
+          author: action.author,
+        },
+      ];
+    case DELETE:
+      return state.filter((book) => book.id !== action.id);
+    default: return state;
+  }
+}
+
+export function createBook(book) {
+  return {
+    type: ADD,
+    book,
+  };
+}
```

Comment:
* Well done working on the requested changes by the previous CR but you forgot to Create and Export also Action Creators for deleting a book. Kindly add it 🙏 [Check requirements ](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=array%20of%20books.-,Export%20Action%20Creators%20for%20your%20actions.,-Write%20your%20reducer)